modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,98 @@
+= Portal
+
+`Unit ability`
+
+Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
+This can be an effective method to deploy reserve forces at the optimum time and place.
+
+Well-known examples include Necron portals and Eldar Webway portals.
+
+---
+
+[NOTE]
+====
+This ability lets you keep detachments outside of the play area at the start of the game, similar to Reserves.
+You can then deploy them later in the game.
+When they arrive, you disembark them into the play area through a unit -- or objective -- that has this ability.
+====
+
+== Data sheet variations of {portal}
+
+Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, in brackets.
+For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.
+
+Mostly, this indicates what other friendly units can actually use the portal -- that is, which kinds of unit you can embark/disembark through that portal during the game.
+
+* `_(INF)_` allows Infantry units to use the {portal}.
+* `_(4+)_` allows Vehicle units that have an Armour value of `4+` or lower to use the {portal}.
+* `_(Walker)_` allows Vehicle units that have the {walker} ability to use the {portal}, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that use it:
+
+* `_(Auto)_` means that detachments in Portal Reserves do *not* follow the standard rules for Reserves.
+Instead, from turn two onwards, you must *must* disembark them from a {portal} unit/objective in the Movement phase unless there are not enough portals available, or none of the available portals can disembark all of the units in the detachment.
+* `_(Roll)_` means that detachments in Portal Reserves follow the standard rules for Reserves.
```

Comment:
_`(Auto)`_ and _`(Roll)`_ are particular to specific {portal} units -- but I can't see how the respective 'follow/don't follow standard Reserves rules' effects could each apply to only some detachments and not others. Is this a gap in the current logic, or am I missing something?

Reply:
The quick answer is that they would never appear in the same list!

So _`(Auto)`_ covers the Necrons who are sitting waiting on the other end of the Portal, while _`(Roll)`_ is future-proofing for Eldar making their way through the Webway.

Reply:
Aaaaah, I see... I didn't get that at all. Okay, we should tweak things a bit such that it is plain that these attributes are in effect mutually exclusive and reflect different technologies in different armies. I'll play around with that.